kitsune/settings.py

```diff
@@ -475,15 +457,9 @@ def path(*parts):
     'commonware.middleware.NoVarySessionMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
 
-    # This should come before TokenLoginMiddleware, because
-    # TokenLoginMiddleware uses this to tell users they have been
-    # automatically logged. It also has to come after
-    # NoVarySessionMiddleware.
+    # This has to come after NoVarySessionMiddleware.
     'django.contrib.messages.middleware.MessageMiddleware',
 
-    # This middleware should come after AuthenticationMiddleware.
-    'kitsune.users.middleware.TokenLoginMiddleware',
```

Comment:
Whats the reason to remove this?

Reply:
This was a mistake, restored. Nice catch!